dDatabase/MigrationRunner.cpp

```diff
@@ -34,10 +34,19 @@ Migration LoadMigration(std::string path) {
 void MigrationRunner::RunMigrations() {
 	Database::Get()->CreateMigrationHistoryTable();
 
+	// has to be here because when moving the files to the new folder, the migration_history table is not updated so it will run them all again.
+	if (!Database::Get()->IsMigrationRun("dlu/mysql/17_migration_for_migrations.sql")) {
+		LOG("Running migration: dlu/mysql/17_migration_for_migrations.sql");
+		Database::Get()->ExecuteCustomQuery("UPDATE `migration_history` SET `name` = CONCAT(SUBSTR(`name`, 1, 4),\"mysql/\", SUBSTR(`name`, 5));");
```

Comment:
Can this get a where clause that checks it only applies when not starting with mysql already?

Also, is the slash always forward when checking against this?

Reply:
ill add some kind of check for this, idk about the slashes though.  ill make it so the slash mimics the one already there with more substr

Reply:
im instead going to invert this and remove paths from consideration altogether.  shouldn't be any confusion there some the migrations are on a per-database case, the file path doesnt matter

Reply:
Please check this code again, it has been updated